tests/cartesian_tests/definitions.py

```diff
@@ -60,3 +61,13 @@ def _get_backends_with_storage_info(storage_info_kind: str):
 @pytest.fixture()
 def id_version():
     return gt_utils.shashed_id(str(datetime.datetime.now()))
+
+
+def _get_array_library(backend: str):
```

Comment:
```suggestion
def get_array_library(backend: str):
```
let's make it public as it's used across modules